cobalt/loader/image/animated_image_tracker.cc

```diff
@@ -17,6 +17,7 @@
 #include <algorithm>
 #include <utility>
 
+#include "base/metrics/histogram_functions.h"
```

Comment:
Shouldn't I see an associated BUILD.gn change for this new #include?

Reply:
No, because //cobalt/loader package already has deps on //base

Reply:
Ah, thought there would be /base/metrics, but makes sense there isn't.